apps/system/utils/netcmd_ping.c

```diff
@@ -193,8 +194,13 @@ nu_ping_recv(int s, struct timespec *ping_time)
 
 	while (1) {
 		len = recvfrom(s, buf, sizeof(buf), 0, (struct sockaddr *)&from, (socklen_t *)&fromlen);
-		if (len <= 0) break;
-
+		if (len < 0) {
+			printf("nu_ping_recv: error [%d]\n", errno);
+			break;
+		}
+		else if (len == 0) {
```

Comment:
This is a coding rule violation. You should locate ```else if``` at same line of closing bracket.
```
} else if (len == 0) {
```

Reply:
Fixed